app/javascript/app/components/country-ghg-map/country-ghg-map-selectors.js

```diff
@@ -146,10 +147,22 @@ export const getPathsWithStyles = createSelector([getDataParsed], data => {
   });
 });
 
-export const getLegendData = createSelector(getYearSelected, year => ({
-  title: `GHG Emissions per capita in ${year}`,
-  buckets
-}));
+export const getLegendData = createSelector(
+  [getCalculationSelection, getYearSelected],
+  (calculation, year) => {
+    let calculationText = '';
+    if (calculation === 'PER_CAPITA') {
+      calculationText = 'per capita ';
+    } else if (calculation === 'PER_GDP') {
```

Comment:
Do you need this? Why not set it as the default and only do one if statement?

Reply:
We have one value that is the empty value. When it's not PER_CAPITA or PER_GDP.

Reply:
I agree, we could use this: https://github.com/Vizzuality/climate-watch/blob/fix/countries-ghg-map-update-legend/app/javascript/app/components/country-ghg-emissions/country-ghg-emissions-selectors.js#L41 a global variables so we don't have strings comparisons. 
what about export a const with the values ?